src/config/config.ts

```diff
@@ -623,11 +629,6 @@ export class Config implements IConfig {
       .map((alias) => [alias.replaceAll('@', '').replaceAll(/[/-]/g, '_'), k].join('_').toUpperCase())
   }
 
-  public scopedEnvVarTrue(k: string): boolean {
```

Comment:
Removing this is a breaking change, which I'd like to avoid

What's the benefit of `scopedEnvVarBoolean`? Is it just so you can use the `??` syntax?

Reply:
Yes, it was to allow using it with ??
When a boolean env var isn't set we might also want to do something. I was using it, but after doing your recommended changes it is no longer necessary. I will revert these changes later.

Reply:
fixed